projects/core/src/model/order.model.ts

```diff
@@ -147,6 +148,7 @@ export interface Order {
   orderDiscounts?: Price;
   orgCustomer?: B2BUser;
   paymentInfo?: PaymentDetails;
+  permissionResults?: OrderApprovalPermissionResult[];
```

Comment:
Should we use _type augmentation_ for that only in scope of the my org lib? @dunqan

Reply:
we have such a plan: https://github.com/SAP/spartacus/issues/8876